src/api/index.ts

```diff
@@ -75,6 +76,7 @@ api.post('/skip', async (req, res) => {
   res.send(await getJudgeTeams(judge));
 });
 
-api.post('/sendUpdateMessage', sendUpdateMessage);
+api.use('/sendUpdateMessage', sendUpdateMessage);
```

Comment:
You can just add the adminMiddleware here to lock down this route, instead of adding it inside of the Router. Makes it easier to see which are locked down and which aren't at a quick glance.

```suggestion
api.use('/sendUpdateMessage', adminMiddleware, sendUpdateMessage);
```